README.md

```diff
@@ -1,2 +1,72 @@
-# gridify
-A tool used to deploy projects using threefold grid 3 
+# Gridify
+
+A tool used to deploy projects using threefold grid 3.
+
+## Build
```

Comment:
build info should be in the very bottom, here should be how to download the latest and use